frontend/public/components/node.tsx

```diff
@@ -152,7 +152,7 @@ const NodeGraphs = requirePrometheus(({node}) => {
       <div className="col-md-12 col-lg-4">
         <Area
           title="Memory Usage"
-          humanize={humanizeDecimalBytes}
+          humanize={humanizeBinaryBytes}
```

Comment:
Won't this cause odd numbers in the y-axis ticks?

Reply:
Why would it?

Reply:
Have we tested?

I'd assume Victory calculates the ticks as round decimal values. If we humanize using binary bytes, I'd guess we'd get values like 0.931 GiB instead of 1 GB, etc. But we'd have to test to see.

Reply:
I haven't tested this to see the output, no.

Unfortunately I haven't been able to get a console up and running myself (I'm new 🙂). @knowncitizen or @rawagner would you be available to take a look?

Reply:
@spadgett is right, y-axis has values with fraction digits